code/game/objects/items/weapons/tools/simple_weapons.dm

```diff
@@ -113,13 +111,24 @@
 	attack_verb = list("slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
 	slot_flags = SLOT_BELT
 	structure_damage_factor = STRUCTURE_DAMAGE_BLADE
+	var/backstab_damage = 5
+
+/obj/item/weapon/tool/knife/apply_hit_effect(atom/target, blocked=FALSE, mob/user)
+	..()
+	if(iscarbon(target) && target.dir == user.dir)
```

Comment:
Good catch @MSieghardt .
Implement a check for forced facing so you don't get front-stabbed Dark Souls 1 style.